index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <!-- Start: Header -->
+    <div class="wrapper">
+      <header class="header">
+        <a href="#">My Logo</a>
+        <nav><i class="fa-solid fa-bars"></i></nav>
+      </header>
+     <!-- End: Header -->
+
+     <!-- Start: Main-->
+      <section>
+        <h1>I’m Hamza Baig. Glad to see <br> you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <ul class="social-media">
+          <li><a href="#"><img src="images/twitter icon.svg" alt="Twitter icon"></a></li>
+          <li><a href="#"><img src="images/linkedin icon.svg" alt="Linkedin icon"></a></li>
+          <li><a href="#"><img src="images/m icon.svg" alt="M icon"></a></li>
+          <li><a href="#"><img src="images/github icon.svg" alt="Github icon"></a></li>
+          <li><a href="#"><img src="images/peace icon.svg" alt="Peace icon"></a></li>
+        </ul>
+      </section>
```

Comment:
Great work incorporating the suggestions from the previous reviewer! I do **have a few optional suggestions that could make your project look even better and more closely match** the [Figma template provided](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=NcRw8dFhHkmW6sux-0):

- [ ] **[OPTIONAL]** Would you mind checking the `padding` you added to your `wrapper`? You could remove it from there and instead apply `padding` **for each section separately**. For example, you could add some `padding` for the `header` and some `padding` for your section. This small adjustment will help your **background look more similar to the template**. 😉

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236096285-8b05a199-f82a-40dc-be96-9bc9e317daa6.png)

  ### Figma design:
  ![image](https://user-images.githubusercontent.com/107952724/236096354-3a2bbe3f-3b93-447d-891d-1b29200bead9.png)

- [ ] **[OPTIONAL]** Additionally, you could try adding `align-items: center` to center the medium icon. This **will help make the design more visually appealing and professional**. 🎯

  ![image](https://user-images.githubusercontent.com/107952724/236096665-1460680b-aa11-45dc-8b82-8ab14500aae1.png)